baseband/gsb/tests/test_gsb.py

```diff
@@ -436,6 +441,12 @@ def test_raw_stream(self, tmpdir):
             assert fh_r.header0.time == fh_r.start_time
             assert fh_r.header0 == frame1.header
             assert fh_r.size == 10 * fh_r.samples_per_frame
+            # Sample rates aren't exactly equal because GSB timestamps only
+            # store 9 digits of the fractional second.
+            assert np.all(np.isclose(fh_r.sample_rate, sample_rate,
```

Comment:
See comment above; thinking further about this: this will screw up actual times and thus make VLBI nearly impossible -- we do need to be sure this is right...  I wonder if we should remove auto-guessing for now and do that correctly in another PR.